src/components/RecipeCard.js

```diff
@@ -1,14 +1,41 @@
 import styled from "styled-components";
+import { useState } from "react";
+import arrowRightIcon from "../assets/icons/ep_arrow-right.svg";
+import arrowDownIcon from "../assets/icons/ep_arrow-down.svg";
 
 export default function RecipeCard({ recipeData }) {
-  const { label, image, totalTime, url } = recipeData.recipe;
+  const { label, image, totalTime, url, healthLabels, ingredients } =
+    recipeData.recipe;
+  const [moreInformation, toggleMoreInformation] = useState(false);
+
+  function toggleInfo() {
+    toggleMoreInformation(!moreInformation);
+  }
 
   return (
     <StyledArticle>
       <h4>{label}</h4>
-      <StyledImg src={image} height="200px" width="250px" alt="" />
-      <p>{totalTime === 0 ? "Time: no data" : `Time: ${totalTime}min`}</p>
-      <a href={url}>See whole recipe</a>
+      <StyledImg src={image} height="250px" width="80%" alt="" />
+      <StyledInfoButton onClick={() => toggleInfo()}>
+        <img src={moreInformation ? arrowDownIcon : arrowRightIcon} alt="" />
+        More information
+      </StyledInfoButton>
+      <StyledInfoSection
+        style={moreInformation ? { display: "unset" } : { display: "none" }}
+      >
+        <StyledTime>{totalTime === 0 ? "" : `~${totalTime}min`}</StyledTime>
+        <ul>
+          <p>Ingredients:</p>
+          {ingredients.map((ingredient, index) => (
+            <li key={index}>{ingredient.text}</li>
+          ))}
+        </ul>
+        <details>
+          <summary>All considered intolerances etc.</summary>
+          <p>{healthLabels.map((label) => ` ${label}`).join(", ")}</p>
```

Comment:
Make this sense? All `label` in one `p` tag?

Reply:
Yeah, I'm not completely happy with this part, because there are so many labels - I just wanted them to take as less space as possible 😅